### ISSUE TITLE
Social Media Connect Buttons Not Working in Staging Environment

### ISSUE THREAD

**Issue by Reshzera** [Giveth/giveth-dapps-v2#4493]
When attempting to connect any social media account (Twitter, Discord, LinkedIn) using the provided buttons on the "Personal Social Media" page, the buttons do not respond and the connection process does not initiate. This issue is occurring in the staging environment. This form is part of the project verification process.

Steps to Reproduce:

1. Navigate to the "Personal Social Media" page in the project verification section in the staging environment.
2. Click on any of the "Connect to X (Twitter)", "Connect to Discord", or "Connect to LinkedIn" buttons.
3. Observe that nothing happens and the social media account is not 
connected.

Expected Behavior:
Clicking on any of the social media connect buttons should initiate the connection process and allow the user to connect their social media accounts.

Actual Behavior:
Clicking on the social media connect buttons does not trigger any action, and the social media accounts are not connected.

![Screenshot 2024-07-29 at 15 02 07](https://github.com/user-attachments/assets/90767dbd-9c6d-4fd7-938b-b02a3a270f64)

## AC
To be tested ONLY on staging -> https://staging.giveth.io/
- [ ] on existing/incomplete GIVbacks form users can connect their DISCORD through the available network buttons (linkedin/discord/twitter) 
- [ ] on existing/incomplete GIVbacks form users can connect their LINKEDIN through the available network buttons (linkedin/discord/twitter) 
- [ ] on existing/incomplete GIVbacks form users can connect their TWITTER through the available network buttons (linkedin/discord/twitter) 
- [ ] on NEW GIVbacks form users can connect their DISCORD through the available network buttons (linkedin/discord/twitter) 
- [ ] on NEW GIVbacks form users can connect their LINKEDIN through the available network buttons (linkedin/discord/twitter) 
- [ ] on NEW GIVbacks form users can connect their TWITTER through the available network buttons (linkedin/discord/twitter) 
- [ ] user can successfully submit givbacks form on both new and existing givbacks form

**Comment by Reshzera** [comment]
Hey @divine-comedian, check if this is happening to you too please

**Comment by divine-comedian** [comment]
I can't even get that far - when I click "Verify Project" from the actions menu on the "My Projects" page for a project I own I get to the 1st page of the verification form - When I click next on the form I receive a client side error 


![image](https://github.com/user-attachments/assets/5d441343-6d8a-4fed-bb73-81f226f5127a)
![image](https://github.com/user-attachments/assets/e3125ee9-d900-45bd-8e16-27b2de309a30)

on this URL/project -> https://staging.giveth.io/verification/testing-the-success-view?tab=verify
logged in as `0x826976d7C600d45FB8287CA1d7c76FC8eb732030` on gnosis chain

**Comment by divine-comedian** [comment]
I tried another project that had already an unsubmitted, but already started verification form - when I tried to connect my linkedin it responded but with an error

![image](https://github.com/user-attachments/assets/43231c41-fae9-4388-b1cd-9456372f7ce9)

when I tried to remove my already added social profile I receive an error in the console and nothing happens 

![image](https://github.com/user-attachments/assets/a077c3dd-4b92-4bc7-8772-dd75075b7bfc)

**Comment by divine-comedian** [comment]
I think this issue is in your court @CarlosQ96 - but also @RamRamez might have some insight - was there any recent code changes that might have broken this feature?

**Comment by divine-comedian** [comment]
@CarlosQ96 @RamRamez what is the update on this please

**Comment by divine-comedian** [comment]
@lovelgeorge99 can check this one out

**Comment by lovelgeorge99** [comment]
the issue here is with the env variables..its value for redirect URL is set same as staging...i will ask devops to change it @divine-comedian

**Comment by divine-comedian** [comment]
Weird... @lovelgeorge99 keep us informed when it can be tested on staging 

when it passes staging QA we should be able to close this issue

**Comment by divine-comedian** [comment]
@lovelgeorge99 is there an update on this issue?

**Comment by lovelgeorge99** [comment]
Hey @divine-comedian  yes ,the devops didn't have access to the social media accounts and I talked to @laurenluz and she said you or @oyealmond might help me with getting access to these accounts .

**Comment by lovelgeorge99** [comment]
@divine-comedian for staging i have created new accounts for twitter and discord and changed the connection URI and now they are working .. but for LinkedIn it doesn't allow me to connect using a personal account it has to be  verified by company's social account...
![image](https://github.com/user-attachments/assets/4f04d1ed-61c6-40f0-9b1d-43dfe476645a)

and to verify it through my account I should verification link to page admin
can you tell me whom to send this link for verification?

**Comment by divine-comedian** [comment]
Thanks @lovelgeorge99 please coordinate with @oyealmond since she has access to Giveth's linkedin account

**Comment by oyealmond** [comment]
hey @lovelgeorge99 , I can make you an admin of the Linkedin Giveth page. I just need your Linkedin profile

**Comment by divine-comedian** [comment]
@lovelgeorge99 if you have time can we come back around to this issue and close it out?

**Comment by lovelgeorge99** [comment]
@divine-comedian this is ready for testing but can be tested locally only as staging is using production BE

**Comment by divine-comedian** [comment]
@lovelgeorge99 we've reverted staging FE settings to use development back-end! I'll move this to QA

**Comment by divine-comedian** [comment]
@maryjaf do you mind taking a look at this one and testing the AC

**Comment by maryjaf** [comment]
On which environment I should  test this issue? @lovelgeorge99

**Comment by maryjaf** [comment]
I've tested it on staging, and the issue related to connecting Discord, X, and LinkedIn has been fixed on staging. @lovelgeorge99 
![image](https://github.com/user-attachments/assets/80ebe624-fd1c-4ad9-87a9-ab839ff9bf59)

However, after connecting one account, the user is redirected to the next step. Is this behavior acceptable? @divine-comedian 

https://github.com/user-attachments/assets/ccee3a91-e43f-4f6f-b7d3-287fdff3e8c8

**Comment by divine-comedian** [comment]
good catch MJ! Let's follow this up in a new issue, indeed after connecting an account the user should stay on the same step of the givbacks form.
